tests/src/JIT/opt/FastTailCall/FastTailCallCandidates.cs

```diff
@@ -0,0 +1,1150 @@
+                                      seven,
+                                      eight,
+                                      new HFASize16(a, b));
+        }
+        else
+        {
+            double b = one + 1599;
+            double a = one + 16;
+            return CalleeWithStackHFA(one, 
+                                      a,
+                                      b,
+                                      four,
+                                      five,
+                                      six,
+                                      seven,
+                                      eight,
+                                      new HFASize16(a, b));
+        }
+    }
+
+    ////////////////////////////////////////////////////////////////////////////
+    // Stack Based args.
+    ////////////////////////////////////////////////////////////////////////////
+
+    public struct StructSizeEightNotExplicit
+    {
+        public long a;
+
+        public StructSizeEightNotExplicit(long a)
+        {
+            this.a = a;
+        }
+    }
+
+    public struct StructSizeSixteenNotExplicit
+    {
+        public long a;
+        public long b;
+
+        public StructSizeSixteenNotExplicit(long a, long b)
+        {
+            this.a = a;
+            this.b = b;
+        }
+
+    }
+
+    /// <summary>
+    /// Possible to fast tail call
+    /// </summary>
+    /// <remarks>
+    ///
+    /// Caller has 6 register arguments and 1 stack argument (size 16)
+    /// Callee has 6 register arguments and 2 stack arguments (size 16)
+    ///
+    /// See: https://github.com/dotnet/coreclr/issues/12468
+    ///
+    /// Return 100 is a pass.
+    /// Return 106 is a failure.
+    ///
+    /// </remarks>
+    public static int CalleeGithubIssue12468(int one,
+                                             int two,
+                                             int three,
+                                             int four,
+                                             int five,
+                                             int six,
+                                             int seven,
+                                             int eight,
+                                             StructSizeEightNotExplicit s1,
+                                             StructSizeEightNotExplicit s2)
+    {
+        int count = 0;
+        for (int i = 0; i < s1.a; ++i)
+        {
+            if (i % 10 == 0)
+            {
+                ++count;
+            }
+        }
+
+        if (count == 160)
+        {
+            return 100;
+        }
+
+        else
+        {
+            return 106;
+        }
+    }
+
+    /// <summary>
+    /// Possible to fast tail call
+    /// </summary>
+    /// <remarks>
+    ///
+    /// Caller has 6 register arguments and 1 stack argument (size 16)
+    /// Callee has 6 register arguments and 2 stack arguments (size 16)
+    ///
+    /// See: https://github.com/dotnet/coreclr/issues/12468
```

Comment:
> /// See: https://github.com/dotnet/coreclr/issues/12468 [](start = 3, length = 56)

Generally, I think you need to explain the case in comments, and not simply refer to a link, which might disappear in the future.